UnitTests/GitCommandsTests/Git/RevisionDiffProviderTest.cs

```diff
@@ -88,6 +88,43 @@ public void RevisionDiffProvider_normal2(string from, string to)
         {
             _revisionDiffProvider.Get(from, to).Should().Be("\"123456789\" \"HEAD\"");
         }
+
+        [TestCase("123456789", GitRevision.UnstagedGuid, "a.txt", null, true)]
+        public void RevisionDiffProvider_fileNameTracked1(string from, string to, string fileName, string oldFileName, bool isTracked)
+        {
+            _revisionDiffProvider.Get(from, to, fileName, oldFileName, isTracked).Should().Be("\"123456789\"  -- \"a.txt\"");
+        }
```

Comment:
I presume tests pass, but to me it all looks very cryptic...
It would be good if you could either rename the test methods to be more descriptive of their nature (e.g. `Get_should_diff_file_from_to_if_file_istracked(...)`) or add some comments describing the test cases.

Reply:
will do